src/fal/run_scripts.py

```diff
@@ -66,13 +66,12 @@ def _prepare_exec_script(script: FalScript, project: FalProject) -> bool:
 
     try:
         script.exec(context, faldbt)
-    except Exception as err:
-        err_str = str.join("\n", traceback.format_exception(err.__class__, err, None))
+    except:
```

Comment:
Does this result in stderr?

Reply:
Do we exit in the end with an error exit code? Or do we just print errors and pretend to succeed?

Reply:
Right now we just print errors and pretend to succeed. But this is the current behavior. We could change it in a different PR.

Reply:
cool